### PR TITLE
Improve "gardenctl target view" formatting

### PR DESCRIPTION
**Pull request by rrhubenov**
**What this PR does / why we need it**:
Improve the format of the "gardenctl target view" subcommand output for better readability.

Before:
```
garden:"local", project:"local", shoot:"local"
```

After:
```
garden: local
project: local
shoot: local
```
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```feature user
NONE
```

### CODE REVIEW

File: pkg/target/target.go [hunk 1]
```diff
@@ -161,30 +161,30 @@ func (t *targetImpl) String() string {
...
 		steps = append(steps, "control plane targeted")
 	}
 
 	if len(steps) == 0 {
-		return "<empty>"
+		return "<empty>\n"
 	}
 
-	return strings.Join(steps, ", ")
+	return strings.Join(steps, "\n") + "\n"
```
Comment (petersutter) [inline pkg/target/target.go:187]: as alternative, we could set the `yaml` output as default, as we do [here](https://github.com/gardener/gardenctl-v2/blob/d889ebb88b103b3cb7695fc66682907af06e5706/pkg/cmd/config/view.go#L53-L55) and get rid of the `String()` implementation as there is currently not much difference
We already have the `yaml` output as default for the `gtv` alias https://github.com/gardener/gardenctl-v2/blob/d889ebb88b103b3cb7695fc66682907af06e5706/pkg/env/templates/rc.tmpl#L11
We could then also omit the `-o yaml` addition on the alias

Comment (rrhubenov) [reply pkg/target/target.go]: Hi Peter, thanks for the suggestion! 
I agree that setting yaml as the default output format is better. I've reworked the 'target view' command to be similar to 'config view' and I've removed the I've removed the `String()` method.

File: pkg/env/templates/rc.tmpl [hunk 2]
```diff
@@ -23,12 +23,12 @@ if [ -z "$GCTL_SESSION_ID" ] && [ -z "$TERM_SESSION_ID" ]; then
   export GCTL_SESSION_ID=$(uuidgen)
 fi
 alias {{.prefix}}=gardenctl
-alias {{.prefix}}tv='gardenctl target view -o yaml'
```
Comment (petersutter) [inline pkg/env/templates/rc.tmpl:26]: the templates for the other shells also have to be adapted

```bash
cat pkg/env/templates/rc.tmpl | grep -n 'gardenctl target view -o yaml'
6:alias {{.prefix}}tv='gardenctl target view -o yaml'
58:alias {{.prefix}}tv='gardenctl target view -o yaml'
79:  gardenctl target view -o yaml
```

File: pkg/env/templates/rc.tmpl [hunk 3]
```diff
@@ -23,12 +23,12 @@ if [ -z "$GCTL_SESSION_ID" ] && [ -z "$TERM_SESSION_ID" ]; then
...
+alias {{.prefix}}tv='gardenctl target view'
 alias {{.prefix}}tc='gardenctl target control-plane'
 alias {{.prefix}}tc-='gardenctl target unset control-plane'
 alias {{.prefix}}k='eval "$(gardenctl kubectl-env {{.shell}})"'
 alias {{.prefix}}p='eval "$(gardenctl provider-env {{.shell}})"'
-alias {{.prefix}}cv='gardenctl config view -o yaml'
```
Comment (petersutter) [inline pkg/env/templates/rc.tmpl:31]: there are still some occurences here as well

```bash
cat pkg/env/templates/rc.tmpl | grep -n 'gardenctl config view -o yaml'
11:alias {{.prefix}}cv='gardenctl config view -o yaml'
63:alias {{.prefix}}cv='gardenctl config view -o yaml'
99:  gardenctl config view -o yaml
```

**Review by petersutter — APPROVED** [review]
/lgtm, though I'd not rename it to `TargetViewOptions` as `target.ViewOptions` is clear and concise

**Review by grolu — APPROVED** [review]
/lgtm

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**